### DISCUSSION TITLE
Appbar.toolbar_opacity always get an error

### DISCUSSION THREAD

**Discussion by AlyAbdelaal** [flet-dev/flet#4277] [General]
in documentation toolbar_opacity of The AppBar is in range from 0 to 1
but it always get an error "toolbar_opacity is out of range (0-1)"
I found that in flet_core/app_bar.py in line 183 the condition is written ```0 >= value >= 1``` so it always out of rang
thats in :
```
@toolbar_opacity.setter
    def toolbar_opacity(self, value: OptionalNumber):
        assert value is None or (
            0 >= value >= 1
        ), "toolbar_opacity is out of range (0-1)"
        self._set_attr("toolbarOpacity", value)
``` 
is that an mistake or there is an other reason behind that

**Comment by ndonkoHenri** [comment]
Thanks for reporting, it will be fixed: https://github.com/flet-dev/flet/issues/4279
